### PR TITLE
Added a Signal Strength Function

### PR DESCRIPTION
**Pull request by graemerae**
Added a function to return Signal Strength
(note previous PR was mangled because of an indent issue on my browser)

### CODE REVIEW

**Comment by dherrada** [comment]
So, the indentation looks good, but it's failing black. I'd start by installing black with pip. Then, run this in the repository folder:
`black --target-version=py35 .`

File: adafruit_rockblock.py [hunk 1]
```diff
@@ -190,7 +190,15 @@ def status(self):
             status = resp[1].strip().decode().split(":")[1]
             return tuple([int(a) for a in status.split(",")])
         return (None,) * 6
-
+      
+    @property
+    def signalStrength(self):
```
Comment (caternuson) [inline adafruit_rockblock.py:195]: Rename to `signal_quality` to avoid use of camel case and better match parameter name to ISU AT Command Reference.

**Review by caternuson — CHANGES_REQUESTED** [review]

**Comment by caternuson** [comment]
@graemerae Keep trying. This is a really good PR to get more familiar with the general PR process. If you commit and push the changes to your PR branch, the updates will automatically be seen here and the checks will run again. No need for a new separate PR.

**Comment by kattni** [comment]
@graemerae Are you still interested in working on this?

**Comment by graemerae** [comment]
If i can figure out how to do a pull request, yes.

**Comment by kattni** [comment]
@graemerae You can push to this PR branch and it will update this PR. I'm guessing you did it through GitHub though, which makes editing it a little more interesting. If you'd like to, you can make the changes and make a new PR with all the changes. Or, we have a guide on Contributing with Git and GitHub [here](https://learn.adafruit.com/contribute-to-circuitpython-with-git-and-github) that might help you figure out running things locally. If you're not already on the [Adafruit Discord](https://adafru.it/discord), we're also available there for questions.

**Comment by caternuson** [comment]
Closing. This was added with #15 and released with 1.2.0. Look for the new `signal_quality` property.

**Pull request closed without merging**